src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (num) => {
```

Comment:
Consider adding a check to ensure that `num` is a number before adding it to `sum`. This will prevent potential errors if a non-numeric value is passed to the adder function.